ethportal-api/src/types/cli.rs

```diff
@@ -154,6 +154,16 @@ pub struct TrinConfig {
     )]
     pub disable_poke: bool,
 
+    #[arg(long = "ws", help = "Used to enable WebSocket rpc.")]
+    pub ws: bool,
+
+    #[arg(
+        long = "ws-port", 
+        help = "The WebSocket port to listen on.", 
+        default_value_t = DEFAULT_WEB3_WS_PORT
```

Comment:
I think we should add [`requires`](https://docs.rs/clap/latest/clap/struct.Arg.html#method.requires) the `ws` argument here